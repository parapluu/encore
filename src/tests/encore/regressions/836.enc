active class Main
  def main() : unit
    println("{}", Nothing)
  end
end
