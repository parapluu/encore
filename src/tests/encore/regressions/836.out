Nothing
0
Nothing
0
5
5
